packages/tooltip/src/Tooltip.tsx

```diff
@@ -11,26 +11,57 @@ import { useMergeRefs } from '@lidofinance/hooks'
 import { isElement } from 'react-is'
 import { TooltipProps } from './types'
 
+const KEEP_TIMEOUT_DEFAULT = 150
+
 function Tooltip(props: TooltipProps, ref?: ForwardedRef<HTMLDivElement>) {
-  const { children, title, ...rest } = props
+  const {
+    title,
+    keepWhileHovering,
+    keepTimeout = KEEP_TIMEOUT_DEFAULT,
+    children,
+    ...rest
+  } = props
+
   const [state, setState] = useState(false)
+  const keepTimeoutRef = useRef(-1)
```

Comment:
you can use `null` instead of "magic" number `-1`

Reply:
Ok